Gemfile

```diff
@@ -123,8 +123,6 @@ group :test do
   gem "rspec-rails", "~> 4.0.1"
   gem 'pickle'
   gem 'shoulda'
-  gem 'capybara', '~> 2.16.1'
-  gem 'cucumber', '~> 3.1'
```

Comment:
I think I'd prefer it if you deleted just the version numbers here, instead of the whole lines. They're included either way, but it's nice to be able to look at the Gemfile and get an idea of which gems are being used.